plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/GroupedProductSelectorItemCTA.php

```diff
@@ -31,12 +31,16 @@ class GroupedProductSelectorItemCTA extends AbstractBlock {
 	 * @return string The HTML markup for the quantity selector.
 	 */
 	private function get_quantity_selector_markup( $product ) {
-		ob_start();
-
-		woocommerce_quantity_input( AddToCartWithOptionsUtils::get_quantity_input_args( $product ) );
+		$args             = AddToCartWithOptionsUtils::get_quantity_input_args( $product );
+		$args['input_id'] = AddToCartWithOptionsUtils::get_quantity_input_id( $product );
 
+		ob_start();
+		woocommerce_quantity_input( $args );
 		$quantity_html = ob_get_clean();
 
+		// Remove the label because we are rendering one as a separate block via GroupedProductSelectorItemLabel.
+		$quantity_html = AddToCartWithOptionsUtils::remove_quantity_label( $quantity_html );
```

Comment:
As we will be always deal with the default `global/quantity-input.php` template, this looks safe to use.